test/postgresql.autoupdate.test.js

```diff
@@ -419,77 +417,7 @@ describe('autoupdate', function() {
         },
       };
 
-      const customer3_schema = {
-        'name': 'CustomerTest3',
-        'options': {
-          'idInjection': false,
-          'postgresql': {
-            'schema': 'myapp_test',
-            'table': 'customer_test3',
-          },
-        },
-        'properties': {
-          'id': {
-            'type': 'String',
-            'length': 20,
-            'id': 1,
-          },
-          'name': {
-            'type': 'String',
-            'required': false,
-            'length': 40,
-          },
-          'email': {
-            'type': 'String',
-            'required': true,
-            'length': 40,
-          },
-          'age': {
-            'type': 'Number',
-            'required': false,
-          },
-        },
-      };
-
       const orderTest_schema_v1 = {
-        'name': 'OrderTest',
-        'options': {
-          'idInjection': false,
-          'postgresql': {
-            'schema': 'myapp_test',
-            'table': 'order_test',
-          },
-          'foreignKeys': {
-            'fk_ordertest_customerId': {
-              'name': 'fk_ordertest_customerId',
-              'entity': 'CustomerTest3',
-              'entityKey': 'id',
-              'foreignKey': 'customerId',
-            },
-          },
-        },
-        'properties': {
-          'id': {
-            'type': 'String',
-            'length': 20,
-            'id': 1,
-          },
-          'customerId': {
-            'type': 'String',
-            'length': 20,
-            'postgresql': {
-              'columnName': 'customerId',
-            },
-          },
-          'description': {
-            'type': 'String',
-            'required': false,
-            'length': 40,
-          },
-        },
-      };
-
-      const orderTest_schema_v2 = {
```

Comment:
very very nitpick(😓 ): I find `orderTest_schema_v1,3,4` but not `orderTest_schema_v2`, can we rename 3 to 2, and 4 to 3 to have a consistent number?